### DISCUSSION TITLE
[灌水]是我不懂现代艺术?

### DISCUSSION THREAD

**Discussion by ybbgh** [share121/inter-knot#3140] [其他]
![47577f75-9f98-4b48-a7ae-6b91b11f0b18](https://github.com/user-attachments/assets/2d567f73-3fc7-45fb-89a8-8566971cf9e5)
最近去看展。旧都陷落时，芭菜大厦被空洞侵蚀的程度还没有现在这么深。空洞调查员紧急抢救出一批展品。
看完只想说，调查员冒着生命危险抢救这种东西，真的有必要吗?
《抱邦布的男人(们)》《以骸背上的芭菜双子》是我最不喜欢的作品，感觉只是戏仿名作，顺便拍下芭菜老板的马屁。作者是金斯顿·伯德。据说这个人没有经过学院派的训练，只是一个临摹名画的画工。
这都可以吗?

**Comment by dolokong** [comment]
有一个说法说这些作品是有原型的，而且都是旧时代的艺术品，不过也无从考证了，毕竟旧时代的艺术品基本全都没保存下来

**Comment by waerte-yang** [comment]
你冤枉金斯顿·伯德啦哈哈哈。
他没有接受过系统的美术训练，青年时期在复制画工厂工作，每天临草名画12个小时，上班上崩溃了，所以恶作剧了一下，在名画里加入了一些无厘头元素。
结果画作落入了芭莱兄弟手中，芭莱兄弟惊为天人，想要联系他。结果他对画廊经理说“买我画的人品味也太差了吧”。
我觉得他很清楚自己画的画就是垃圾，不过能够狠狠地宰芭莱兄弟两个冤大头，就是
爽啦哈哈哈<img src="https://img-static.mihoyo.com/communityweb/upload/59353722c9fb0c5d926e84638b044e02.gif" alt="爱了爱了" width="75" >
